Formula/ansible@2.0.rb

```diff
@@ -83,114 +83,114 @@ class AnsibleAT20 < Formula
   # Install requests with [security]
   #
   resource "cffi" do
-    url "https://pypi.python.org/packages/source/c/cffi/cffi-1.5.2.tar.gz"
+    url "https://files.pythonhosted.org/packages/source/c/cffi/cffi-1.11.tar.gz"
     sha256 "da9bde99872e46f7bb5cff40a9b1cc08406765efafb583c704de108b6cb821dd"
   end
```

Comment:
The way the "latest" version, v1.11 isn't released yet anywhere, it is `(NOT RELEASED SO FAR)` is a little odd...